woodwork/tests/schema/test_table_schema.py

```diff
@@ -159,7 +159,7 @@ def test_schema_equality_standard_tags(sample_column_names, sample_inferred_logi
     assert schema != no_standard_tags_schema
 
     # Different standard tags values - no logical types that have standard tags
-    nl_ltypes = {col_name: NaturalLanguage for col_name in sample_column_names}
+    nl_ltypes = {col_name: Unknown for col_name in sample_column_names}
```

Comment:
Maybe we should rename this variable to `unknown_types` instead of `nl_types`?

Reply:
done!